content/writing/blog-posts.md

```diff
@@ -23,6 +23,12 @@ When writing a blog, use your outline as a guide and expand on all points as nee
 
 We are looking forward to this and are excited to read what you write!!!
 
+### Writing tool

```

Comment:
This should be a header 2 (##) just like the others

Reply:
Actually, it's a section inside "Writing your blog post" so I think H3 is fine. I should convert the last section to H3 also.